i18n/en.json

```diff
@@ -367,6 +371,10 @@
     "id": "api.channel.rename_channel.cant_rename_group_messages.app_error",
     "translation": "You cannot rename a group message channel"
   },
+  {
+    "id": "api.channel.undelete_channel.unarchived",
```

Comment:
You forgot to also reorder this one. see https://build.mattermost.com/blue/organizations/jenkins/mp%2Fmattermost-server-pr-new/detail/PR-13349/14/pipeline

Reply:
😭 . done.